packages/earn-controller/src/EarnController.ts

```diff
@@ -503,31 +669,485 @@ export class EarnController extends BaseController<
     address,
   }: RefreshPooledStakingDataOptions = {}): Promise<void> {
     const errors: Error[] = [];
+    for (const chainId of this.#supportedPooledStakingChains) {
+      await Promise.all([
+        this.refreshPooledStakes({ resetCache, address, chainId }).catch(
+          (error) => {
+            errors.push(error);
+          },
+        ),
+        this.refreshStakingEligibility({ address }).catch((error) => {
+          errors.push(error);
+        }),
+        this.refreshPooledStakingVaultMetadata(chainId).catch((error) => {
+          errors.push(error);
+        }),
+        this.refreshPooledStakingVaultDailyApys(chainId).catch((error) => {
+          errors.push(error);
+        }),
+        this.refreshPooledStakingVaultApyAverages(chainId).catch((error) => {
+          errors.push(error);
+        }),
+      ]);
+    }
+
+    if (errors.length > 0) {
+      throw new Error(
+        `Failed to refresh some staking data: ${errors
+          .map((e) => e.message)
+          .join(', ')}`,
+      );
+    }
+  }
+
+  /**
+   * Refreshes the lending markets data for all chains.
+   * Updates the lending markets in the controller state.
+   *
+   * @returns A promise that resolves when the lending markets have been updated
+   */
+  async refreshLendingMarkets(): Promise<void> {
```

Comment:
It looks like we've added some new methods to the controller, what do you think about noting them in the changelog?